src/screens/NewsScreen/index.js

```diff
@@ -238,10 +239,16 @@ const NewsScreen = () => {
           onScroll={handleScrollEvent}
           scrollEventThrottle={16}
           data={news}
-          refreshing={refreshing}
-          onRefresh={onRefresh}
+          refreshControl={
+            <RefreshControl
+              tintColor={COLORS.white2}
+              refreshing={refreshing}
+              onRefresh={onRefresh}
+            />
+          }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A `RefreshControl` component has been added to the `FlatList` component. This enables pull-to-refresh functionality in the list. The `tintColor` prop sets the color of the refresh indicator. Make sure the color value provided is consistent with the app's color scheme. Also, ensure that the `refreshing` prop is correctly managed in the state to indicate whether the list is currently refreshing or not, and the `onRefresh` function properly handles the refresh action.
```diff
+          refreshControl={
+            <RefreshControl
+              tintColor={COLORS.white2}
+              refreshing={refreshing}
+              onRefresh={onRefresh}
+            />
+          }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->